src/axis.ts

```diff
@@ -239,12 +251,12 @@ export interface Axis extends AxisConfigBaseWithConditional, Guide {
    *
    * __Default value__: `undefined`
    */
-  tickMinStep?: number;
+  tickMinStep?: number | SignalRef; // Vega already supports Signal
```

Comment:
What does the comment mean?

Reply:
But it is not a part of BaseAxis that we derive from.

Reply:
We should change Vega-typings such that Axis's extension could be refered in VL, but I don't have a good name for the extension (...) below.

```
export interface Axis extends BaseAxis {
  ...
}
```

In any case, I think that's beyond the scope of this PR.